functests/test_resources/estate_info_story.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# The step field is a string that represents the method bound to an endpoint and
+# the args and kwargs correspond to the arguments to such a method.
+#
+# NB the assets and events endpoints require all values to be strings. Other values may
+# be of the correct type such as confirm which is a boolean.
+#
+# This story Evaluates how many assets and events exist.
+steps:
+  - step:
+      action: ASSETS_COUNT
+      description: Count all assets
+      print_response: true
+
+  - step:
+      action: EVENTS_COUNT
+      description: Count all events
+      print_response: true
+
+  - step:
+      action: LOCATIONS_COUNT
+      description: Count all locations
+      print_response: true
+
+  - step:
```

Comment:
I don't think we really need this. The double-check method and duplicate finder were there from the very early days when it was possible to pick up double records from blockchain caches and things. That doesn't happen anymore, so I think just counting the 3 things is enough. The question comes though, how do we pretty-print the answers/count numbers?